splinterd/src/daemon/mod.rs

```diff
@@ -88,21 +84,24 @@ use splinter::rest_api::auth::authorization::AuthorizationHandler;
 use splinter::rest_api::auth::authorization::Permission;
 #[cfg(feature = "oauth")]
 use splinter::rest_api::OAuthConfig;
-use splinter::rest_api::{
-    AuthConfig, Method, Resource, RestApiBuilder, RestApiServerError, RestResourceProvider,
-};
+use splinter::rest_api::{AuthConfig, Method, Resource, RestApiBuilder, RestResourceProvider};
 use splinter::service;
 #[cfg(feature = "service-arg-validation")]
 use splinter::service::validation::ServiceArgValidator;
 use splinter::threading::lifecycle::ShutdownHandle;
 use splinter::transport::{
-    inproc::InprocTransport, multi::MultiTransport, AcceptError, ConnectError, Connection,
-    Incoming, ListenError, Listener, Transport,
+    inproc::InprocTransport, multi::MultiTransport, AcceptError, Connection, Incoming, Listener,
+    Transport,
 };
 
 use crate::node_id::get_node_id;
 use crate::routes;
-use crate::UserError;
+
+pub mod builder;
```

Comment:
By convention modules should be added at the top of the file, before the use statements